landscape/client/monitor/tests/test_computertags.py

```diff
@@ -0,0 +1,52 @@
+from landscape.client.monitor.computertags import ComputerTags
+from landscape.client.tests.helpers import MonitorHelper, LandscapeTest
+
+
+class ComputerTagsTest(LandscapeTest):
+
+    helpers = [MonitorHelper]
+
+    def setUp(self):
+        super(ComputerTagsTest, self).setUp()
+        self.plugin = ComputerTags()
+        self.monitor.add(self.plugin)
+
+    def test_tags_are_read(self):
+        """
+        Tags are read from the default config file
```

Comment:
tags should be read from the currently used config file, not just the default one.

Reply:
I added a test case to cover this